docs/contributing.rst

```diff
@@ -259,7 +259,7 @@ from the documentation or the other tests:
 
 The AFL documentation states that the corpus (the initial input files) should not be
 too large. The files themselves should not be larger than 1 kB and there should be
-at most one input file per functionality, so better start with a small number of
+at most one input file per functionality. There is also a tool called ``afl-cmin`` that can trim input files
```

Comment:
I think these are two different comments.

Reply:
I would not recommend it, we do not know if it works for our case.